### ISSUE TITLE
Sizes of training set and test set in LineMOD

### ISSUE THREAD

**Issue by kevinsanity5** [j96w/DenseFusion#190]
I noticed that in the LineMOD dataset, the size of training set is much smaller than the test set (accoding to `train.txt` and `test.txt`). But in `dataset.py`, you actually take about 10% of the 'test set' as validation set. Is that right? 
May I ask the reason why the sizes of the training set, the validation set and the test set are assigned this way?
Thank you!

**Comment by lunalulu** [comment]
hello, i have the same question, do you understand that?
